cluster/manifests/kube-node-decommissioner/01-rbac.yaml

```diff
@@ -16,6 +16,9 @@ metadata:
     application: kubernetes
     component: kube-node-decommissioner
 rules:
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["list"]
 - apiGroups: [""]
   resources: ["nodes"]
   verbs: ["list", "patch"]
```

Comment:
We likely need `update` instead of `patch` here :facepalm:  can you also add `update` just to be on the safe side.

Reply:
Changed!